docs/source/blueprints/BP004.rst

```diff
@@ -0,0 +1,211 @@
+BP004
+#####
+
+:Number: BP004
+:Title: Dependency graph
+:Author: Willian Rampazzo <willianr@redhat.com>
+:Discussions-To: https://github.com/avocado-framework/avocado/pull/5125
+:Reviewers:
```

Comment:
Maybe you could add the reviews at this point.

Reply:
Sure, I can do that when I add the issues.